appveyor.yml

```diff
@@ -7,16 +7,29 @@ assembly_info:
   assembly_file_version: "{version}"
   assembly_informational_version: "{version}"
 platform: x64
+environment:
+  nuget_user:
+    secure: kWtlS49bKmP0IFXmdvDHmxGI8mS0xQGrwiJoco9nYtA=
+  nuget_password:
+    secure: Pzvx2CQjxp8eCBP+nFI2eg==
```

Comment:
I am not so sure - we have to pass them somehow, instead of hardcoding them in plaintext in scripts. These are password which allow read-only access to MATLAB NuGet package & to connect to build worker through RDP *during the build*.